presto-main/src/main/java/com/facebook/presto/Session.java

```diff
@@ -709,6 +709,7 @@ public Session build()
         private Optional<Duration> executionTime = Optional.empty();
         private Optional<Duration> cpuTime = Optional.empty();
         private Optional<DataSize> peakMemory = Optional.empty();
+        private Optional<DataSize> peakTaskTotalMemory = Optional.empty();
```

Comment:
let's just call it `peakTaskMemory` to aligned with `peakMemory` :) . Ditto for other variable/strings